linode/manager
# CI: Publish Storybook

## Description

**What does this PR do?**
This publishes storybook on merges to master.

This we should consider doing later:
- Adding a workflow step to run Storybook unit tests between the "build" and "publish" steps (need to write tests first)
- Deploying to a staging site on merges to `staging`
- Deploying to a dev site on merges to `develop`
- Versioning? Not really sure how that would work

## How to test

**What are the steps to reproduce the issue or verify the changes?**

You can see a successful run of the workflow here: 
https://github.com/johnwcallahan/manager/runs/5242741475

**How do I run relevant unit tests?**
